File: my-solutions/0070-climbing-stairs/solution.py
class Solution:
    def climbStairs(self, n: int) -> int:
        if n == 0 or n == 1:
            return 1
        else:
            n0 = 1
            n1 = 1
            n2 = -1
            for i in range(2, n+1):
                n2 = n1 + n0
                n0 = n1
                n1 = n2
            return n2

File: my-solutions/0070-climbing-stairs/solution.cpp
class Solution {
public:
    int climbStairs(int n) {
        int* res = new int[n + 1];
        res[1] = 1;
        res[0] = 1;
        for(int i = 2; i < n + 1; ++i) {
            res[i] = res[i - 1] + res[i - 2];
        }
        int result = res[n];
        delete[] res;
        return result;
    }
};
